lib/feature_management.rb

```diff
@@ -34,7 +34,7 @@ def self.development_and_telephony_test_adapter?
   end
 
   def self.prefill_otp_codes_allowed_in_sandbox?
-    Identity::Hostdata.domain == 'identitysandbox.gov' && telephony_test_adapter?
+    Identity::Hostdata.domain.end_with?('identitysandbox.gov') && telephony_test_adapter?
```

Comment:
Minor risk, but we should probably do something so that abcidentitysandbox.gov doesn't work

Reply:
I didn't think that this was really going to be a problem, because the real thing that we are looking for is that it's not login.gov, but if you do, I'll put that in.  :-)